diff --git a/dist/node/chunk-Zsoi3j4v.js b/dist/node/chunk-Zsoi3j4v.js
index 8cfe39e30b9e866566716751deb57a58555b7bf8..87d80cff5751d63590810e8bdeab2fcba94c2899 100644
--- a/dist/node/chunk-Zsoi3j4v.js
+++ b/dist/node/chunk-Zsoi3j4v.js
@@ -40586,7 +40586,7 @@ async function localSearchPlugin(siteConfig) {
   };
 }
 const headingRegex = /<h(\d*).*?>(.*?<a.*? href="#.*?".*?>.*?<\/a>)<\/h\1>/gi;
-const headingContentRegex = /(.*?)<a.*? href="#(.*?)".*?>.*?<\/a>/i;
+const headingContentRegex = /(.*)<a.*? href="#(.*?)".*?>.*?<\/a>/i;
 function* splitPageIntoSections(html) {
   const result = html.split(headingRegex);
   result.shift();
